src/openseadragon.js

```diff
@@ -690,13 +690,6 @@
   */
 
 
- /**
```

Comment:
@avandecreme I think you were the one who found that we can't document `OpenSeadragon` as both a function and a namespace? We still had this doc comment (for the function) lying around, but it wasn't a problem before because it didn't recognize it as a function (it was a complicated `window.OpenSeadragon = ...` construct before). Now that it looks like a function again, it caused the namespace not to be documented, including all of the options!

Anyway, I'm pretty sure we just need this doc comment gone; the existence of this function is well documented in the home page of the docs and in the constructor for Viewer.